client/src/pages/profile/profile.js

```diff
@@ -11,14 +12,22 @@ import Spinner from '../../components/Shared/Spinner/Spinner.js'
 import './Profile.scss'
 import '../../styles/rubberDucky.scss'
 
-const Profile = ({ auth: { user, loading }, mode, toggleJournalConfigSetting, getJournalConfig, journalConfig }) => {
+const Profile = ({ auth: { user, loading }, mode, toggleJournalConfigSetting, getJournalConfig, journalConfig, isAuthenticated }) => {
   useEffect(() => {
     getJournalConfig()
   }, [])
 
-  if (loading) {
-    return <Spinner />
+  if (!isAuthenticated) {
+    console.log(isAuthenticated)
+    return <Redirect to="/login" />;
```

Comment:
This logic will overlap with some stuff I've already got going on so we can remove this too